database/seedMethods.js

```diff
@@ -0,0 +1,181 @@
+const {createData} = require('./sampleDataModel');
+const Promise = require('bluebird');
+
+//creates a data array for 100 BOOKS
+var createDataArray = () => {
+  let dataArray = [];
+  for (var i = 0; i < 100; i++) {
+    var data = createData();
+    dataArray.push(data);
+  }
+  return dataArray;
+};
+
+//SEEDS ONE data object to database.
+var seedDb = (data, db) => {
+
+  let details = data.mainDetails;
+  let characters = data.characters;
+  let awards = data.litAwards;
+  let editions = data.editions;
+
+  //details Table Seeder Function
+  var seedDetailsTable = (details) => {
+    let { type, pageNum, publisher, title, dates, isbn10, isbn13, language } = details;
+    let queryString = 'INSERT INTO details (type, pagenum, publisher, firstPubDate, originalPubDate, title, isbn10, isbn13, language) values (?, ?, ?, ?, ?, ?, ?, ?, ?);';
```

Comment:
nice attempt, but if the table comments above have been reconciled, you may need to revisit this seeder and ensure that only required fields are seeded

Reply:
👍 I think I will take a good look at what information might be redundant. However, I'm pretty sure the isbn numbers, and publication dates, even publishers are all different for every edition.